app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<%# in app/views/new.html.erb %>
+
+<%= form_with model: @task, class: 'create-task' do |f| %>
+  <p>Please provide the following information to save your task to our database:</p>
+
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+
+  <%= f.label :completed_at %>
+  <%= f.text_field :completed_at %>
```

Comment:
Consider whether this value should be input in the form or set as a default to `nil`.